resources/timezones/README.md

```diff
@@ -0,0 +1,40 @@
+# Timezone Mapper
+
+The timezone mapper allows one to get a likely timezone for a particular phone
+number. The timezone returned is the canonical ID from [CLDR](
+http://www.unicode.org/cldr/charts/latest/supplemental/zone_tzid.html), not a
+localised name (or any other identifier). For mobile phones which are associated
+with particular area codes, it returns the timezone of the area code; it does
+not track the user's current location in any way. This could be used to work out
+whether it is likely to be a good time to ring a user based on their provided
+number.
+
+Code Location:
+[java/geocoder/src/com/google/i18n/phonenumbers/PhoneNumberToTimeZonesMapper.java](https://github.com/googlei18n/libphonenumber/blob/master/java/geocoder/src/com/google/i18n/phonenumbers/PhoneNumberToTimeZonesMapper.java)
+
+Example usage:
+
+```
+PhoneNumberToTimeZonesMapper timeZonesMapper = PhoneNumberToTimeZonesMapper.getInstance();
+
+List<String> timezones = timeZonesMapper.getTimeZonesForNumber(phoneNumber);
+```
+
+## Contributing to the timezone metadata
+
+The timezone metadata is auto-generated with few exceptions, so we cannot accept
+pull requests. If we have an error please file an issue and we'll see if we can
+make a generic fix.
```

Comment:
This is what we've said when explaining that timezone metadata is auto-generated: https://github.com/googlei18n/libphonenumber/issues/1265, https://github.com/googlei18n/libphonenumber/issues/1131, https://github.com/googlei18n/libphonenumber/issues/1133

It's different for timezone metadata (and maybe certain regions' geocoding) than any other bugs, because it's about making an automatic fix that fixes the issues and doesn't change anything else. So the "fix" is to the process, not the metadata; and it has to be generic, not about the given issue.